dasharo-security/measured-boot.robot

```diff
@@ -301,8 +301,10 @@ Measured Boot Suite Setup
     Prepare Test Suite
     Skip If    not ${MEASURED_BOOT_SUPPORT}    Measured boot is not supported
     Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    Tests in Ubuntu are not supported
+    Skip If    '${TPM_EXPECTED_VERSION}' == '0'    TPM Version in platform config does not support this suite
```

Comment:
Why not `Skip If    not ${TPM_SUPPORT}`? Alternatively, both config options could be merged, with `TPM_EXPECTED_VERSION` equal to 0 (or even `none` because of dynamic typing) meaning what `TPM_SUPPORT = False` currently is used for.